playbooks/capsule/tasks/main.yaml

```diff
@@ -0,0 +1,65 @@
+---
+- name: 'Set SELinux to Permissive'
+  selinux: policy=targeted state=permissive
+
+- name: 'Get Server Hostname'
+  shell: hostname -f
+  delegate_to: "{{ server }}"
+  register: server_fqdn
+- name: 'Get Server IP Address'
+  shell: ifconfig eth0 2>/dev/null|awk '/inet/ {print $2}'|sed 's/addr://'
+  delegate_to: "{{ server }}"
+  register: server_ip
+- set_fact: server_ip="{{ server_ip.stdout.split('\n')[0] }}"
+
+- name: 'Setup Communication to Capsule'
+  delegate_to: "{{ server }}"
+  shell: echo "{{ capsule_ip }} {{ ansible_nodename }}" >> /etc/hosts
```

Comment:
http://docs.ansible.com/ansible/lineinfile_module.html would be probably preferred here, to keep the playbook idempotent